src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -1043,6 +1052,15 @@ public void verifyDefaultImageUrl(String imgSrc) {
         browser.closeCurrentWindowAndSwitchToParentWindow();
     }
 
+    /**
+     * Returns if the input element is valid (satisfies constraint validation).
+     */
+    public boolean isInputElementValid(WebElement inputElement) {
+        Preconditions.checkArgument(inputElement.getAttribute("nodeName").equals("INPUT"));
```

Comment:
I think a static import for common preconditions checks would be reasonable I believe?